src/changefile/promptForChange.ts

```diff
@@ -29,20 +28,19 @@ export async function promptForChange(options: BeachballOptions) {
 
     const disallowedChangeTypes = getDisallowedChangeTypes(pkg, packageInfos, packageGroups);
     const packageInfo = packageInfos[pkg];
-    const showPrereleaseOption = prerelease(packageInfo.version);
     const changeTypePrompt: prompts.PromptObject<string> = {
       type: 'select',
       name: 'type',
       message: 'Change type',
       choices: [
-        ...(showPrereleaseOption ? [{ value: 'prerelease', title: ' [1mPrerelease[22m - bump prerelease version' }] : []),
```

Comment:
I'm not sure that enabling prerelease by default will be desirable for everyone. One possible workaround is to add a separate option for always allowing prerelease, though that increases option bloat and is less discoverable for people who do want the feature. @kenotron any thoughts?